README.md

```diff
@@ -0,0 +1,74 @@
+# Methods for Estimating the Union of Multiple Sets
```

Comment:
I see a few spelling errors in here, my apologies, will make sure to fix those before submission

Reply:
I think I got them all, but please let me know if you find any